library/core/src/macros/panic.md

```diff
@@ -5,9 +5,9 @@ to the caller of the program. `panic!` should be used when a program reaches
 an unrecoverable state.
 
 This macro is the perfect way to assert conditions in example code and in
-tests. `panic!` is closely tied with the `unwrap` method of both [`Option`]
-and [`Result`][runwrap] enums. Both implementations call `panic!` when they are set
-to None or Err variants.
+tests. `panic!` is closely tied with the `unwrap` method of both
+[`Option`][ounwrap] and [`Result`][runwrap] enums. Both implementations call
+`panic!` when they are set to None or Err variants.
 
 This macro is used to inject panic into a Rust thread, causing the thread to
 panic entirely. Each thread's panic can be reaped as the `Box<Any>` type,
```

Comment:
What do you think, is it too busy?
```suggestion
panic entirely. Each thread's panic can be reaped as the [`Box`]`<`[`Any`]`>` type,
```

Reply:
I think I'm for it; people may think that `Any` is a placeholder (like the generic `T` in `Box<T>`) rather than a concrete trait. (People like myself ;)